### PR TITLE
 Enable inline refactoring for private values 

### PR DESCRIPTION
**Pull request by sschaef**
I don't know why it was disabled for private values, seems to be an
oversight to me.

Fixes #1002648

### CODE REVIEW

File: src/main/scala/scala/tools/refactoring/implementations/InlineLocal.scala [hunk 1]
```diff
@@ -26,10 +26,12 @@ abstract class InlineLocal extends MultiStageRefactoring with ParameterlessRefac
       case None => s.findSelectedOfType[ValDef]
     }
 
+    def isInliningAllowed(sym: Symbol) =
+      (sym.isPrivate || sym.isLocal) && !sym.isMutable && !sym.isValueParameter
+
     selectedValue match {
-      case Some(t) if (t.symbol.isPrivate || t.symbol.isLocal)
-        && !t.symbol.isMutable && !t.symbol.isValueParameter
-        && t.symbol.enclMethod != NoSymbol => Right(t)
```
Comment (wpopielarski) [inline src/main/scala/scala/tools/refactoring/implementations/InlineLocal.scala]: not sure how it worked previously, but seems like to be limited to method bodies only? (just question)

**Comment by wpopielarski** [comment]
LGTM

File: src/main/scala/scala/tools/refactoring/implementations/InlineLocal.scala (hunk 1 above)
Comment (sschaef) [inline src/main/scala/scala/tools/refactoring/implementations/InlineLocal.scala]: yes, it was limited to method bodies but I don't understand why. Also, the condition checks for `sym.isPrivate` which can never be true for local values. This makes the code even more confusing.

Comment (misto) [inline src/main/scala/scala/tools/refactoring/implementations/InlineLocal.scala]: Agreed, that didn't make any sense. Maybe the behavior was different back in 2.8, but I don't really remember.

**Pull request merged**